### PR TITLE
Bump up to Hotfix Version

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

**Review by heaths — CHANGES_REQUESTED** [review]
IMO, this may indeed be a practical fix, but shouldn't we bump versions as needed so we don't have to remember to change them back to 0 (and bump the minor or major) when merging/cherry-picking back into master? We had to do this manually during previews. Perhaps an upfront check to make sure we don't collide with published packages *before* we try to publish (which, at least for nuget.org and npm I know, would fail) would be better?

File: sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>Azure Client SDK integration with Microsoft.Extensions libraries</Description>
     <AssemblyTitle>Azure Client SDK integration Microsoft.Extensions</AssemblyTitle>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.0.1</VersionPrefix>
```
Comment (weshaggard) [inline sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj]: This one did not ship in master when we created this fork so we should undo this until those changes come into this repo.

**Comment by weshaggard** [comment]
>IMO, this may indeed be a practical fix, but shouldn't we bump versions as needed so we don't have to remember to change them back to 0 (and bump the minor or major) when merging/cherry-picking back into master? We had to do this manually during previews. 

In general the flow will be make the change in master and then cherry pick it into the release branch in this case. However no matter which direction you merge there should be a merge conflict on the version property and you should generally always stick with the version that is in the branch and not override it. It should also be noted for this time we were asked to bump them all to help out the library team during this dec-rollup as we expect to have most of them ship in this time.

 >Perhaps an upfront check to make sure we don't collide with published packages before we try to publish (which, at least for nuget.org and npm I know, would fail) would be better?

We do have a check in our release pipeline that will fail if you try to publish version that already exists. We have considered pulling that closer to the PR validation but we don't want to do that until we have some better tooling for version updating.

**Comment by heaths** [comment]
Still, not all packages will release at the same time, so wouldn't leaving servicing branch service bumps in owners' hands be more reflective of their own releases?

File: sdk/core/Azure.Core/src/Azure.Core.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the implementation of the Azure Client Pipeline</Description>
     <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
-    <Version>1.0.0</Version>
+    <Version>1.0.1</Version>
```
Comment (pakrym) [inline sdk/core/Azure.Core/src/Azure.Core.csproj]: This was bumped to 1.0.0-preview.1 3 days ago and now it's 1.0.1 and PR has no description. What exactly is our plan in regards to versioning?

Comment (weshaggard) [reply sdk/core/Azure.Core/src/Azure.Core.csproj]: @pakrym this is the servicing branch, we are trying to go ahead bump the patch version for everything here in preparation for doing a set of roll-up fixes for Dec. For master the version was bumped to `1.1.0-preview.1` which is still the case.

**Comment by weshaggard** [comment]
>Still, not all packages will release at the same time, so wouldn't leaving servicing branch service bumps in owners' hands be more reflective of their own releases?

That is fair we are mostly doing this at the request of the leads. @AlexGhiondea should we have individual owners bump their patch version as necessary?

**Pull request closed without merging**